tests/user_liberty_vars.yml

```diff
@@ -26,10 +26,6 @@ maas_excluded_alarms:
   - '^Network transmit rate.*'
   - '^percentage_disk_utilisation.*'
 
-# swift-recon does not have '--time' option in Liberty
-maas_excluded_checks:
```

Comment:
This may introduced a gate error. See comment below

Reply:
TASK [Install ironic ironic-compute checks] ************************************
Thursday 21 June 2018  15:43:07 +0000 (0:00:00.804)       0:03:04.504 ********* 
fatal: [aio1_ironic_compute_container-f92f0cbb -> 172.29.236.100]: FAILED! => {"changed": false, "failed": true, "msg": "AnsibleUndefinedVariable: 'maas_excluded_checks_regex' is undefined"}

Reply:
@tonytan4ever I'm not sure how the removal of that var is impacting newton ?